source/clear-linux/get-started/virtual-machine-install/vmware-player-install-cl.rst

```diff
@@ -0,0 +1,301 @@
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Set virtualization engine option
+
+      Figure 8: VMware Workstation 14 Player - Set virtualization engine
+      option
+
+#. Under the :guilabel:`Virtualization engine` section,
+   check :guilabel:`Virtualize Intel VT-x/EPT or AMD-V/RVI`.
+
+#. Click the :guilabel:`Close` button.
+
+#. Click the :guilabel:`Finish` button.
+
+Install Clear Linux into the new VM
+***********************************
+
+#. Select the newly-created VM and click the :guilabel:`Play virtual machine`
+   button. See Figure 9.
+
+   .. figure:: figures/vmware-player/vmware-player-install-cl-9.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Power on virtual machine
+
+      Figure 9: VMware Workstation 14 Player - Power on virtual machine
+
+#. Follow the :ref:`install-on-target` guide to complete the installation of
+   |CL|.
+
+#. After the installation completes, reboot the VM. This reboot restarts the
+   |CL| installer.
+
+Detach the |CL| installer ISO from the VM
+*****************************************
+
+#. To enable the mouse pointer so you access VMware Workstation Player's
+   menus, press :kbd:`<CTRL>` + :kbd:`<ALT>` on the keyboard.
+
+#. To disconnect the CD/DVD to stop it from booting the |CL| installer ISO
+   again, click the :guilabel:`Player` menu. See Figure 10.
+
+   .. figure:: figures/vmware-player/vmware-player-install-cl-10.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Edit CD/DVD settings
+
+      Figure 10: VMware Workstation 14 Player - Edit CD/DVD settings
+
+#. Go to :menuselection:`Removable Devices-->CD/DVD (IDE)-->Settings`.
+
+#. On the :guilabel:`Device status` section, uncheck the
+   :guilabel:`Connected` and the :guilabel:`Connect at power on` settings.
+   See Figure 11.
+
+   .. figure:: figures/vmware-player/vmware-player-install-cl-11.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Disconnect CD/DVD
+
+      Figure 11: VMware Workstation 14 Player - Disconnect CD/DVD
+
+#. Click the :guilabel:`OK` button.
+
+#. To power off the VM, click the :guilabel:`Player` menu. See Figure 12.
+
+   .. figure:: figures/vmware-player/vmware-player-install-cl-12.png
+      :scale: 100%
+      :alt: VMware Workstation 14 Player - Power off virtual machine
+
+      Figure 12: VMware Workstation 14 Player - Power off virtual machine
+
+#. Go to :guilabel:`Power` and select :guilabel:`Shut Down Guest`.
+
+Enable UEFI boot support
+************************
+
+|CL| needs UEFI support to boot. To enable UEFI, add the
+following line to the end of your VM's :file:`.vmx` file:
+
+  .. code-block:: bash
+
+    firmware = "efi"
+
+VMware typically saves VM files on the following locations:
+
+* On Linux distros: :file:`/home/username/vmware`
+* On Windows: :file:`C:/\Users/\username/\Documents/\Virtual Machines`
+
+The file type is `VMware virtual machine configuration`.
+
+Power on the VM
```

Comment:
@bktan8 I expected this section to include the steps to power on the VM. Can you add those steps or are they included somewhere else?